### DISCUSSION TITLE
Regarding the scale bar or length data on the image

### DISCUSSION THREAD

**Discussion by ghost** [marksgraham/OCT-Converter#65] [General]
Hello, 

  Thank you so much for creating such helpful package, this really helps me out. 
I had a question regarding the metadata. I was wondering if these images have the information of the length per pixel in the meta data. 
According to the variables that I get, I couldn't find one, but I was hoping I could convert my E2E files into tiff files and apply scale bars on them at once.  
I couldn't find an option on the heidelberg OCT device, so I was hoping that the images itself contained the information.
Thank you!

**Comment by marksgraham** [comment]
Hi,

The bscanmedata here contains some variables that might be of interest
https://github.com/marksgraham/OCT-Converter/blob/380dd63933cfa9f03d4a2d12f44c8ec880e60014/oct_converter/readers/e2e.py#L236

In particular, there is a variable called scaleY which could be the length per pixel? Have you got any ground-truth length per pixels that you might be abe to use to verify if this variable is what you want?
